<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1268</width>
    <height>857</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>421</width>
     <height>361</height>
    </rect>
   </property>
   <widget class="QPushButton" name="connectButton_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_1">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>MCL_nanodrive</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Newport_Conex_microdrive</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="connectButton_1">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>140</y>
      <width>159</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Newport_Conex_microdrive</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MCL_nanodrive</string>
     </property>
    </item>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>401</width>
      <height>101</height>
     </rect>
    </property>
    <widget class="QPushButton" name="z_inc_2">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>10</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>z+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="x_dec_2">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>40</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>x-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="y_dec_2">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>70</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>y-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="z_dec_2">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>70</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>z-</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="xlineEdit_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="ylineEdit_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>38</y>
       <width>46</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>y position</string>
     </property>
    </widget>
    <widget class="QPushButton" name="y_inc_2">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>10</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>y+</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>67</y>
       <width>45</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>z position</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="x_y_inc_lineEdit_2">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>40</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="z_inc_lineEdit_2">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>40</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>10</y>
       <width>46</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>x position</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="zlineEdit_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="confirm_x_button_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>10</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Confirm x</string>
     </property>
    </widget>
    <widget class="QPushButton" name="confirm_y_button_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Confirm y</string>
     </property>
    </widget>
    <widget class="QPushButton" name="confirm_z_button_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Confirm z</string>
     </property>
    </widget>
    <widget class="QPushButton" name="x_inc_2">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>40</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>x+</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_1" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>401</width>
      <height>101</height>
     </rect>
    </property>
    <widget class="QLineEdit" name="ylineEdit_1">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="zlineEdit_1">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>67</y>
       <width>45</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>z position</string>
     </property>
    </widget>
    <widget class="QPushButton" name="confirm_x_button_1">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>10</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Confirm x</string>
     </property>
    </widget>
    <widget class="QPushButton" name="confirm_z_button_1">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Confirm z</string>
     </property>
    </widget>
    <widget class="QPushButton" name="confirm_y_button_1">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Confirm y</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>9</y>
       <width>46</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>x position</string>
     </property>
    </widget>
    <widget class="QPushButton" name="z_dec_1">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>70</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>z-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="y_inc_1">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>10</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>y+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="x_inc_1">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>40</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>x+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="x_dec_1">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>40</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>x-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="y_dec_1">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>70</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>y-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="z_inc_1">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>10</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>z+</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>38</y>
       <width>46</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>y position</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="xlineEdit_1">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="x_y_inc_lineEdit_1">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>40</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="z_inc_lineEdit_1">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>40</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Display:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="display_option">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>280</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>MU300</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="save_button">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>330</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Save_Point</string>
    </property>
   </widget>
   <widget class="QComboBox" name="snapshot_live_comboBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>280</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Live</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Snapshot</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="Sample_Selector_comboBox">
    <property name="geometry">
     <rect>
      <x>318</x>
      <y>280</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>New Sample</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Old Sample</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="Point_Selector_comboBox">
    <property name="geometry">
     <rect>
      <x>98</x>
      <y>310</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>NV</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bottom left</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bottom right</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Top left</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Top right</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>280</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Sample Selector:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Point Selector:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Find_NV_Button">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>330</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Find_NV</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="File_name_lineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>340</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>File_name:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
